apps/pwabuilder/src/script/components/app-package-form-base.ts

```diff
@@ -275,25 +275,35 @@ export class AppPackageFormBase extends LitElement {
   }
 
   private colorChanged(e: UIEvent, formInput: FormInput) {
-    const inputElement = e.target as HTMLInputElement | null;
-    let formattedValue = (inputElement as unknown as SlColorPicker)!.getFormattedValue('hex').toLocaleUpperCase();
+    interface HTMLSlColorPicker extends HTMLInputElement,  SlColorPicker {
```

Comment:
@Jaylyn-Barbee I did like this. But it's weird that the inputElement is treated as two different types at same time.